utils/dc-chain/patches/arm-Darwin/gcc-9.3.0-kos.diff

```diff
@@ -0,0 +1,1527 @@
+diff -ruN gcc-9.3.0/gcc/config.host gcc-9.3.0-kos/gcc/config.host
+--- gcc-9.3.0/gcc/config.host	2020-03-12 05:07:21.000000000 -0600
++++ gcc-9.3.0-kos/gcc/config.host	2022-10-03 15:12:44.000000000 -0600
+@@ -93,8 +93,8 @@
+ case ${host} in
+   *-darwin*)
+     # Generic darwin host support.
+-    out_host_hook_obj=host-darwin.o
+-    host_xmake_file="${host_xmake_file} x-darwin"
++    # out_host_hook_obj=host-darwin.o
++    # host_xmake_file="${host_xmake_file} x-darwin"
+     ;;
+ esac
+
+diff -ruN gcc-9.3.0/gcc/config/host-darwin.c  gcc-9.3.0-kos/gcc/config/host-darwin.c
+--- gcc-9.3.0/gcc/config/host-darwin.c	2020-03-12 05:07:21.000000000 -0600
++++ gcc-9.3.0-kos/gcc/config/host-darwin.c	2022-10-03 15:13:15.000000000 -0600
+@@ -22,6 +22,8 @@
+ #include "coretypes.h"
+ #include "diagnostic-core.h"
+ #include "config/host-darwin.h"
++#include "hosthooks.h"
++#include "hosthooks-def.h"
+ 
+ /* Yes, this is really supposed to work.  */
+ static char pch_address_space[1024*1024*1024] __attribute__((aligned (4096)));
+@@ -75,3 +77,5 @@
+ 
+   return ret;
+ }
++
++const struct host_hooks host_hooks = HOST_HOOKS_INITIALIZER;
+
```

Comment:
Everything below this point is an exact copy of /utils/dc-chain/patches/gcc-9.3.0-kos.diff